pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarColumnHandle.java

```diff
@@ -58,6 +59,16 @@
 
     private final Integer[] positionIndices;
 
+    /**
+     * True if the column is key column handler for KeyValueSchema.
+     */
+    private final boolean key;
+
+    /**
+     * True if the column is value column handler for KeyValueSchema.
+     */
+    private final boolean value;
+
```

Comment:
It's better to use a since enum to indicate the column is key column or value column for KeyValueSchema.

Reply:
I'll use an enum instead to them